src/argilla/client/feedback/schemas/remote/shared.py

```diff
@@ -30,6 +31,7 @@ def _client(self) -> Optional[httpx.Client]:
         return self.client
 
     class Config:
+        # TODO(@alvarobart) Not sure if we need this at this level
```

Comment:
`Config` is inherited AFAIK, but we do need to `arbitrary_types_allowed = True` here because otherwise we cannot add the `httpx.Client` type-hint